plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -853,6 +875,40 @@ public Optional<JdbcTableHandle> getTableHandle(ConnectorSession session, Schema
                         "com.google.common.util.concurrent.UncheckedExecutionException: java.lang.RuntimeException: first attempt is poised to fail");
     }
 
+    @Test
+    public void testSpecificSchemaAndTableCaches()
+    {
+        CachingJdbcClient cachingJdbcClient = createCachingJdbcClient(
+                FOREVER,
+                Duration.succinctDuration(2, SECONDS),
+                Duration.succinctDuration(1, SECONDS),
```

Comment:
Can we increase the difference between the two ttl a bit more higher so as to avoid flakkiness ?